src/test/java/core/basesyntax/service/strategy/BalanceOperationHandlerTest.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.service.strategy;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.storage.FruitStorage;
+import org.junit.jupiter.api.Test;
+
+class BalanceOperationHandlerTest {
+    private static final OperationHandler BALANCE_OPERATION_HANDLER = new BalanceOperationHandler();
```

Comment:
your handler is not a constant. it is the service that you test. Fix it in all test classes. Initialize it in `BeforeAll`